Update formal-review-process.md (#548)
mattermost/mattermost-handbook: operations/workplace/people/performance-reviews-50/formal-review-process.md

## Feedback panel requests

> Managers will select peers in Culture Amp who can most effectively provide feedback on achievements and performance over the past year \(this can include upward feedback for people managers. Depending on the role, PT Staff and community members may be solicited to provide feedback in addition to the peer panel selected in Culture Amp\). Managers should log in to Culture Amp, go to "my team > request team feedback" and determine the peer panels for each direct report. More information for managers on selecting peer panels is available [here](https://academy.cultureamp.com/hc/en-us/articles/360003789699-Request-team-feedback). Prior to selecting peer panels and writing performance reviews for their direct reports, managers should take the Culture Amp training available [here](https://www.cultureamptraining.com/performance-for-managers). 

Reviewer: ```suggestion
Managers will select peers in Culture Amp who can most effectively provide feedback on achievements and performance over the past year \(this can include upward feedback for people managers. Depending on the role, PT Staff and community members may be solicited to provide feedback in addition to the peer panel selected in Culture Amp\). Managers should log in to Culture Amp, go to **my team > request team feedback** and determine the peer panels for each direct report. More information for managers on selecting peer panels is available [here](https://academy.cultureamp.com/hc/en-us/articles/360003789699-Request-team-feedback). 

Prior to selecting peer panels and writing performance reviews for their direct reports, managers should take the Culture Amp training available [here](https://www.cultureamptraining.com/performance-for-managers).
```

## Peer feedback

> Managers will select the peer panels in Culture Amp. To get started, log in to Culture Amp and navigate to your "task list" where you'll find requests for peer or upward feedback. On average, teammates should be completing feedback surveys for 3 - 6 of their peers. 

Reviewer: ```suggestion
Managers will select the peer panels in Culture Amp. To get started, log in to Culture Amp and navigate to your **task list** where you'll find requests for peer or upward feedback. On average, teammates should be completing feedback surveys for 3 - 6 of their peers.
```

## Self reflection

> FT Staff will receive an email via Culture Amp to complete their self-assessment. To get started, log in to Culture Amp and navigate to your "task list" where you'll find a request to complete your Self-reflection. Once your self-reflection has been submitted, managers are able to make comments directly in your self-assessment. Prior to writing their self-reflections, FT Staff should take the Culture Amp training available [here](https://www.cultureamptraining.com/participate-in-a-performance-cycle-for-employees). 

Reviewer: ```suggestion
Full-time staff will receive an email via Culture Amp to complete their self-assessment. To get started, log in to Culture Amp and navigate to your **task list** where you'll find a request to complete your self-reflection. Once your self-reflection has been submitted, managers are able to make comments directly in your self-reflection document. Prior to writing their self-reflections, full-time staff should take the Culture Amp training available [here](https://www.cultureamptraining.com/participate-in-a-performance-cycle-for-employees).
```

## Performance Review

> Once self-reviews and peer feedback have been completed, Managers will receive a notification from Culture Amp inviting them to write the performance reviews.  Managers will read all feedback from the peer panel and self-assessment, and will incorporate their feedback based on their own performance expectations of this role. 

Reviewer: ```suggestion
Once self-reviews and peer feedback have been completed, Managers will receive a notification from Culture Amp inviting them to write the performance reviews. Managers will read all feedback from the peer panel and self-assessment, and will incorporate their feedback based on their own performance expectations of this role.
```

> When the review is complete, Managers will not share the reviews with their reports until Calibration has been completed and any changes have been approved. HR will turn on "sharing" after all results have been finalized. Managers will then share the reviews with their direct reports accordingly. 

Reviewer: ```suggestion
When the review is complete, Managers will not share the reviews with their reports until calibration has been completed and any changes have been approved. HR will turn on "sharing" after all results have been finalized. Managers will then share the reviews with their direct reports accordingly.
```

> *Note: It's helpful to share the review ahead of the 1:1 meeting to allow for time to reach and digest feedback.* 

Reviewer: ```suggestion
*Note: It's helpful to share the review ahead of the 1:1 meeting to allow for time to read and digest feedback.*
```

## Calibration

All people Managers within the same [department](https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-spend-company-money/how-to-use-expensify#departments) or other group that the Team [MLT](https://handbook.mattermost.com/operations/operations/mlt-cadence) indicates will participate in calibrations.

> Managers will share their review 'expectation' ratings (meets, exceeds, does not meet), any recommendations for a [level](https://docs.google.com/document/d/1XNGYOHouoY42YYmFHNrhu-vHDtWVrPL5E8M_BpwF9iU/edit) change and/or a promotion, in the performance review for each of their reports. This recommendation is not shared with reports. HR will prepare Calibration reports for each department/team and will share the report with the calibration group ahead of the meeting. During the meeting, Managers share feedback on any performance metric and apply [high standards](https://handbook.mattermost.com/company/about-mattermost#leadership-principles) in evaluation against the criteria for staff members' role and level.

Reviewer: ```suggestion
Managers will share their review expectation ratings (meets, exceeds, does not meet), any recommendations for a [level](https://docs.google.com/document/d/1XNGYOHouoY42YYmFHNrhu-vHDtWVrPL5E8M_BpwF9iU/edit) change and/or a promotion, in the performance review for each of their reports. This recommendation is not shared with reports. HR will prepare calibration reports for each department/team and will share the report with the calibration group ahead of the meeting. During the meeting, Managers share feedback on any performance metric and apply [high standards](https://handbook.mattermost.com/company/about-mattermost#leadership-principles) in evaluation against the criteria for staff members' role and level.
```
